### PR TITLE
Page creation article

### PR DESCRIPTION
**Pull request by tiwiex**
included some clarity on creating the controller bundle. not a syntax error. The step is hidden in the code while the instruction to extend Symfony Controller with the LuckyController.

### CODE REVIEW

File: page_creation.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ random) number and prints it. To do that, create a "Controller class" and a
     class LuckyController
     {
         /**
-         * @Route("/lucky/number")
+          @Route("/lucky/number")
```
Comment (xabbuh) [inline page_creation.rst]: this change looks wrong

File: page_creation.rst [hunk 2]
```diff
@@ -94,7 +94,10 @@ If you're returning HTML from your controller, you'll probably want to render
 a template. Fortunately, Symfony comes with `Twig`_: a templating language that's
 easy, powerful and actually quite fun.
 
-First, make sure that ``LuckyController`` extends Symfony's base
+First, include the Symfony controller class as shown in line 5 below then,
```
Comment (xabbuh) [inline page_creation.rst]: [...] below. Then [...]

**Review by xabbuh — APPROVED** [review]

**Comment by tiwiex** [comment]
I thought I cancelled it. I am sorry if it came in. My first GIT commit but I didn't complete the pull request. I did a recheck before I completing the process and everything surprisingly worked so I stopped. New to smfony. Actually a Drupal developer and then realized a need to understand how Symfony works. Having composer night mares here. Great job you guys are doing.
I did have issues with this though.origin  git@github.com:YOUR-GITHUB-USERNAME/symfony-docs.git (fetch)
origin  git@github.com:YOUR-GITHUB-USERNAME/symfony-docs.git (push)
upstream  https://github.com/symfony/symfony-docs.git (fetch)
upstream  https://github.com/symfony/symfony-docs.git (push) Had to change origin URL to "https" to get it to work. i.e https://github.com/tiwiex/symfony-docs.gitorigin https://github.com/tiwiex/symfony-docs.git (fetch)origin https://github.com/tiwiex/symfony-docs.git (push)upstream https://github.com/symfony/symfony-docs.git (fetch)upstream https://github.com/symfony/symfony-docs.git (push)
Is this something I could have done differently or a change needs to be made?
Regards,Taiwo
The most terrifying words in the english language are "I'm from the government and I am here to help" - Ronald Reagan 

    On Friday, September 15, 2017 7:48 AM, Christian Flothmann <notifications@github.com> wrote:
 

 @xabbuh commented on this pull request.In page_creation.rst:> @@ -48,7 +48,7 @@ random) number and prints it. To do that, create a "Controller class" and a
     class LuckyController
     {
         /**
-         * @Route("/lucky/number")
+          @Route("/lucky/number")
this change looks wrong—

**Review by javiereguiluz — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @tiwiex.

**Pull request merged**